21.md

```diff
@@ -162,36 +169,36 @@ These examples show all the possible uses of this extension by a supporting `WAL
 // GET bipa.app/callback?amount=538000&convert=BRL
 {
   "pr": "(invoice of 538 sats)",
-  "converted": 1
+  "converted": 100
```

Comment:
should we add that converted is in the smallest currency unit as well?

Reply:
ab98b7d6ce1350dd4e2426eb110e7c6cf216c862